### PR TITLE
Bring preview message in line with Figma

### PR DESCRIPTION
**Pull request by david-mears-dfe**
I spotted a difference between production and the Figma for sprint 50, which was a 5-minute job so I just did it.

## Figma

See 1.8. https://www.figma.com/file/n2wLtKVQ2imMbjdPSHBRKW/Sprint-50?node-id=0%3A1

### Before

Screenshot 2020-06-15 at 17 48 02

### After

Screenshot 2020-06-15 at 18 36 57

## Next steps:

- [x] UX sign-off

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -327,7 +327,7 @@ en:
       button: 'Preview job listing'
       page_title: 'Preview job listing for %{school}'
       summary:
-        heading: 'Preview of %{title}'
+        heading_html: 'Preview of &#8216;%{title}&#8217; job listing'
```
Comment (cpjmcquillan) [inline config/locales/en.yml:330]: `heading: Preview of '%{title}' job listing` if you prefer. We don't use fancy apostrophes anywhere else on the site yet as far as I know

Comment (david-mears-dfe) [reply config/locales/en.yml:330]: I thought the straight apostrophes looked ugly.

Comment (david-mears-dfe) [reply config/locales/en.yml:330]: Screenshot 2020-06-15 at 18 19 59

For your consideration

Comment (cpjmcquillan) [reply config/locales/en.yml:330]: Tomayto tomahto, I'm happy either way if UX/Content are happy

**Pull request merged**